releasenotes/notes/refactor-di-er-enablement-env-var-12056782561eecc6.yaml

```
@@ -0,0 +1,6 @@
+---
+deprecations:
+  - |
+    exception replay: The ``DD_EXCEPTION_DEBUGGING_ENABLED`` environment
+    variable has been deprecated in favor of ``DD_EXCEPTION_REPLAY_ENABLED``.
+    The old environment variable will be removed in a future major release.
```

Should we explicitly specify 3.x here?
```suggestion
    The old environment variable will be removed in the next major release (3.x).
```